app/components/ui/QrCodeModal/QrCodeModal.tsx

```diff
@@ -20,34 +19,28 @@ export const QrCodeModal = ({
   return (
     <Modal
       isOpen={isOpen}
-      className={styles.qrCodeModal}
-      overlayClassName={styles.qrCodeModalOverlay}
-      onRequestClose={closeModal}
+      addModalClass={styles.qrCodeModal}
+      closeModal={closeModal}
     >
-      <div
-        role="button"
-        tabIndex={0}
-        onClick={closeModal}
-      >
-        <img
-          src={icon('close')}
-          alt="close"
-          className={styles.closeModal}
-        />
-      </div>
-      <h2 className={styles.title}>Resource Code</h2>
-      <p className={styles.body}>
-        Scan this QR code with a mobile phone to open this page.
-      </p>
-      <QRCodeSVG value={window.location.href} size={235} />
-      <div>
-        <Button
-          onClick={closeModal}
-          addClass={styles.closeBtn}
-          tabIndex={0}
-        >
-          CLOSE
-        </Button>
+      <div className={styles.modalContent}>
+        <div className={styles.description}>
+          <h2 className={styles.title}>Resource Code</h2>
+          <p className={styles.body}>
+            Scan this QR code with a mobile phone
+            <br />
```

Comment:
Is this necessary? It may be difficult in general to ensure that this breaks at a reasonable place when using different font sizes (remember, the user may want to increase the site font for accessibility) and if the text changes in the future.

When I removed this `<br />` tag locally, it seemed to put a line break at a reasonable place to me.

If we're trying to avoid a line that just has a single word on it, one trick that I learned a long time ago is to put a non-breaking space (`&nbsp;`) between the last and second to last words, which will ensure that the final line will have at least two words.

Reply:
Thanks, good trick! Done.